src/lib/middleware/middleware.js

```diff
@@ -0,0 +1,63 @@
+/**************************************************************************
```

Comment:
Shouldn't this middleware have an appropriate name to indicate that it's for `tracing`?